Microsoft.Azure.CosmosRepository/src/Options/RepositoryOptions.cs

```diff
@@ -1,68 +1,57 @@
-﻿// Copyright (c) IEvangelist. All rights reserved.
-// Licensed under the MIT License.
-
-using Microsoft.Azure.Cosmos;
+﻿// Copyright (c) IEvangelist. All rights reserved. Licensed under the MIT License.
```

Comment:
I'm not sure I like this being formatted this way either, the options were structured in an order that made sense - now it is just alphabetical...which is less ideal for authors.

Reply:
Alphabetical is the default setting in CodeMaid. I just ran the formatter on it... It does some opinionated stuff... Most of the changes in this PR are cosmetic. There are some actual things though. I'll go make a version that is formatted closer to the original and incrementally change things. That should be a bit easier to read.